stm32-modules/include/heater-shaker/heater-shaker/motor_task.hpp

```diff
@@ -360,9 +366,37 @@ class MotorTask {
         }
     }
 
+    // unit test this
+    // comments
+    auto set_homing_speed(
+        std::array<char, SYSTEM_WIDE_SERIAL_NUMBER_LENGTH> serial_number)
+        -> void {
+        int result{};
+        auto [ptr, ec]{std::from_chars(serial_number.begin() + 5,
+                                       serial_number.end(), result)};
+        if (ec == std::errc()) {
+            if (result > 2022113007) {  // make define. is this always true?
```

Comment:
this logic leaves the `else` for this if out - could make it `if (ec == std::errc() && result > 2022113007)` and it'd be fine